app/[locale]/projects/[projectId]/page.tsx

```diff
@@ -0,0 +1,300 @@
+              <div className="">
+                <h2 id="plans-title">{t('plans')}</h2>
+              </div>
+              <div className="actions" role="group"
+                   aria-label={t('planActions')}>
+                <Link
+                  href="/plans/new"
+                  className="react-aria-Button react-aria-Button--secondary"
+                  aria-label={t('uploadPlan')}
+                >
+                  {t('upload')}
+                </Link>
+                <Link
+                  href="/plans/new"
+                  className="react-aria-Button react-aria-Button--primary"
+                  aria-label={t('createNewPlan')}
+                >
+                  {t('createNew')}
+                </Link>
+              </div>
+            </div>
+            {project.plans.map((plan) => (
+              <Card className="plan-item" key={plan.id}>
+                <p className="mb-1">{t('funder')}: {plan.funder_name}</p>
+                <h3 className="mt-0">{plan.template_name}</h3>
+                <div className="plan-sections mb-4">
+                  <ul className="plan-sections-list"
+                      aria-label={t('planSections')}>
+                    {plan.sections.map((section) => (
+                      <li key={section.id} className="plan-sections-list-item">
+                        <Link href={section.link}>
+                          {section.section_title}
+                        </Link>
+                        <span className="plan-sections-list-item-progress">
+                          {t('progress', {
+                            current: section.progress,
+                            total: 3
+                          })}
+                        </span>
+                      </li>
+                    ))}
+                  </ul>
+                </div>
+                <div className="plan-meta">
+                  <p>
+                    {t('doi')}: {plan.doi} <br/>
+                    {t('lastUpdated')}: {plan.last_updated}<br/>
+                    {t('template')}: Arctic Data Center: NSF Polar Programs
+                  </p>
+                </div>
+                <div className="plan-footer">
+                  <div className="plan-links">
+                    <Link
+                      href="/plans/123"
+                      className="plan-link download-link"
+                      aria-label={t('downloadPlan')}
+                    >
+                      <span className="link-text">{t('download')}</span>
+                      <span className="link-icon" aria-hidden="true">↓</span>
+                    </Link>
+                    <Link
+                      href="/share"
+                      className="plan-link share-link"
+                      aria-label={t('sharePlan')}
+                    >
+                      <span className="link-text">{t('share')}</span>
+                      <span className="link-icon" aria-hidden="true">👤</span>
```

Comment:
Left icons effectively empty - we probably need to think what icons are good for these